aldryn_jobs/cms_appconfig.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from aldryn_apphooks_config.models import AppHookConfig
+from cms.models.fields import PlaceholderField
+
+
+class JobsConfig(AppHookConfig):
+    # Job PHFs
+    placeholder_jobs_top = PlaceholderField(
+        'jobs_top', related_name='aldryn_jobs_top')
+
```

Comment:
@intellectronica to be clear, it is customary to divide the imports into "sections" for clarity, separating python imports from Django imports from 3rd party imports from "local" imports.